pyomo/gdp/plugins/bigm.py

```diff
@@ -152,14 +151,15 @@ def __init__(self):
             Set:         False,
             SetOf:       False,
             RangeSet:    False,
-            Disjunction: self._warn_for_active_disjunction,
+            Disjunction: False,
```

Comment:
Do we no longer need to warn for active disjunctions?  I suppose not, as the preprocessing would have identified all Disjunctions on the active block/disjunct tree.  A comment to this effect might be warranted.

Reply:
Yep, exactly, it's unreachable code. In bigm because of going from leaf to root. In hull because we actually handle them as we encounter them. I'll add a comment.